### PR TITLE
HKG: resume from standstill for HDA1 CAN-FD

### PR DESCRIPTION
**Pull request by hoomoose**
**Description**
Resume from Standstill does not work for HKG Can-FD cars with HDA1. This is fixed by putting the can send for Cruise Buttons on bus 6 rather than bus 5. The fix requires an IF statement, which I have not written since I am not sure how to do so.
This fix only solves the problem for HDA1 cars with the 0x1cf signal like the EV6 and I5.
Solves issue: https://github.com/commaai/openpilot/issues/25771

**Prerequisite**
This requires a simple Panda change on L39 of safety_hyundai_canfd.h
Bus 0 needs to be bus 2. That is it.
see: https://github.com/commaai/panda/pull/1102

**Verification** 
This fix has been employed in a fork hoomoose:hda1ev6i5 written specifically for hda1 ev6 and I5 cars and has been successfully tested/used by many!

### CODE REVIEW

**Comment by mattwebbio** [comment]
Also interested in seeing this get merged!

File: selfdrive/car/hyundai/hyundaicanfd.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ def create_buttons(packer, cnt, btn):
     "SET_ME_1": 1,
     "CRUISE_BUTTONS": btn,
   }
-  return packer.make_can_msg("CRUISE_BUTTONS", 5, values)
+  #this needs to be bus 6 for hda1 cars and bus 5 for hda2
+  return packer.make_can_msg("CRUISE_BUTTONS", 6, values)
```
Comment (sshane) [inline selfdrive/car/hyundai/hyundaicanfd.py:49]:
```suggestion
  return packer.make_can_msg("CRUISE_BUTTONS", get_e_can_bus(CP), values)
```

File: selfdrive/car/hyundai/hyundaicanfd.py [hunk 2]
```diff
@@ -39,13 +39,22 @@ def create_cam_0x2a4(packer, camera_values):
...
     "CRUISE_BUTTONS": btn,
   }
-  return packer.make_can_msg("CRUISE_BUTTONS", 5, values)
+  
+  if CP.flags & HyundaiFlags.CANFD_HDA2:
+    ret.append(packer.make_can_msg("CRUISE_BUTTONS", 5, values))
+  else:
+    ret.append(packer.make_can_msg("CRUISE_BUTTONS", 6, values))
+  
+  return ret
```
Comment (sshane) [inline selfdrive/car/hyundai/hyundaicanfd.py:57]:
```suggestion
  if CP.flags & HyundaiFlags.CANFD_HDA2:
    return packer.make_can_msg("CRUISE_BUTTONS", 5, values)
  else:
    return packer.make_can_msg("CRUISE_BUTTONS", 6, values)
```

no need to make it a list, I'd also see if we might want a helper function to identify this bus like we do for e can

File: selfdrive/car/hyundai/hyundaicanfd.py [hunk 3]
```diff
@@ -39,13 +39,22 @@ def create_cam_0x2a4(packer, camera_values):
   })
   return packer.make_can_msg("CAM_0x2a4", 4, camera_values)
 
-def create_buttons(packer, cnt, btn):
+def create_buttons(packer, CP, cnt, btn):
+
+  ret = []
+
```
Comment (sshane) [inline selfdrive/car/hyundai/hyundaicanfd.py:45]:
```suggestion
def create_buttons(packer, CP, cnt, btn):
```

File: selfdrive/car/hyundai/hyundaicanfd.py (hunk 2 above)
Comment (hoomoose) [reply selfdrive/car/hyundai/hyundaicanfd.py:57]: My code didn't work anyways... It gives an error. I will try your code and see if that works for me.

**Comment by hoomoose** [comment]
Current code works!

**Pull request merged**